pygeo/DVGeometry.py

```diff
@@ -2832,7 +2832,8 @@ def _finalize(self):
                     for j in range(self.FFD.vols[iVol].nCtlv):
                         for k in range(self.FFD.vols[iVol].nCtlw):
                             ind = self.FFD.topo.lIndex[iVol][i, j, k]
-                            if coefMask[ind] == False and ind not in self.axis[key]["ignoreInd"]:
+                            # we have to cast to bool to compare using "is"
```

Comment:
... I guess this was never used before, otherwise we would have spotted the broken `if` statement. How did you get this? Anyway, I am fine with the bool() cast

Reply:
Nope, so using `==` worked previously, but it's not the preferred Python syntax:
```
>>> np.array([False]) == False
array([ True])
>>> bool(np.array([False])) == False
True
```

I think I will switch these to `not <statement>` which is more pythonic, and does the `bool` cast automatically.
```
>>> not np.array([False])
True	
```